integration/tests/common/approvalstate/result_approval_state.go

```diff
@@ -12,43 +12,46 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-const resultApprovalTimeout = 100 * time.Second
+const resultApprovalTimeout = 120 * time.Second
```

Comment:
Timeout for result approval is better to set at least twice of the blocking timeout basically since verification nodes only verify results included in a finalized block. So there is at least an interval of two blocks, i.e., a block being finalized + its execution result of it available for verification in another finalized block.